cpp/benchmarks/io/orc/orc_reader_input.cpp

```diff
@@ -72,7 +72,7 @@ void BM_orc_read_data(nvbench::state& state, nvbench::type_list<nvbench::enum_ty
                         data_profile_builder().cardinality(cardinality).avg_run_length(run_length));
   auto const view = tbl->view();
 
-  cuio_source_sink_pair source_sink(io_type::HOST_BUFFER);
+  cuio_source_sink_pair source_sink(io_type::FILEPATH);
```

Comment:
ooh, I didn't fully comprehend the proposed change until now.
I would argue that HOST_BUFFER is a better choice here (device buffer is the best option IMO) because this set of benchmarks is meant to benchmark only the decode part of the reader (the name is "orc_read_decode"). I understand that this leaves us with a gap in benchmark coverage (different types + IO). maybe that should either be a separate benchmark or an option.

Reply:
cc @GregoryKimball